tools/querytee/proxy_endpoint.go

```diff
@@ -117,6 +124,10 @@ func (p *ProxyEndpoint) executeBackendRequests(r *http.Request, resCh chan *back
 				bodyReader = io.NopCloser(bytes.NewReader(body))
 			}
 
+			if b.filter != nil && !b.filter.Match([]byte(r.URL.String())) {
+				return
```

Comment:
I think we should log these, to make it easy to determine why a request was not proxied

Reply:
Done :slightly_smiling_face: